### PR TITLE
Fix header img src field

### PR DESCRIPTION
**Pull request by jhonny218**
## Description
The image's `src` doesn't match the GraphQL build HTML. This is the same in all transformers.

The [header.htm](  <script type="text/javascript">)l template file populates `headerFooterData` which contains the information displayed
```
  <script type="text/javascript">
    window.VetsGov = window.VetsGov || {};
    window.VetsGov.headerFooter = {{ headerFooterData }};
  </script>
```

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ncksllvn** [comment]
I just stumbled upon this PR and IDK what the changes do, but the URLs in that config are absolute because of the header "injection" we do across legacy VA pages/domains

**Comment by jhonnyoddball** [comment]
@ncksllvn Yes, the URLs are absolute, you can see the transformation [here](https://github.com/department-of-veterans-affairs/vets-website/blob/69cb9387b87f1141368f1768f5993f44e16722b1/src/site/stages/build/drupal/menus.js#L152)
```
return {
    img: {
      src: convertLinkToAbsolute(hostUrl, img.derivative.url),
      alt: img.alt || '',
    },
    link: {
      text: link.title,
      href: convertLinkToAbsolute(hostUrl, link.url.path),
    },
    description: promo.entity.fieldPromoLink.entity.fieldLinkSummary,
  };
```

The code only adjusts `img.derivative.url` and that's done in the transformer for the promo content

**Review by Clark-W-Griswold — APPROVED** [review]
👍

File: src/site/stages/build/process-cms-exports/transformers/block_content-promo.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
+const { getImageCrop } = require('./helpers');
+
 const transform = entity => ({
   entity: {
     entityType: 'block_content',
     entityBundle: 'promo',
-    fieldImage: { entity: entity.fieldImage[0] },
+    fieldImage: {
+      entity: getImageCrop(entity.fieldImage[0], '_32MEDIUMTHUMBNAIL'),
```
Comment (U-DON) [inline src/site/stages/build/process-cms-exports/transformers/block_content-promo.js:8]: Where is `_32MEDIUMTHUMBNAIL` coming from? I see it's one of the `mediaImageStyles` in `helpers`, but I'm not sure how we came up with this value in the first place. It also seems like it should be a constant.

Comment (jhonnyoddball) [reply src/site/stages/build/process-cms-exports/transformers/block_content-promo.js:8]: It's coming from the [GraphQL query](https://github.com/department-of-veterans-affairs/vets-website/blob/9757b4454077729edf67e17880c3620a7a9d60b4/src/site/stages/build/drupal/graphql/navigation-fragments/header.nav.graphql.js#L29) from the header

**Pull request merged**